infrastructure/proto/log_replication_metadata.proto

```diff
@@ -25,3 +25,23 @@ message LogReplicationMetadataKey {
 message LogReplicationMetadataVal {
   string val = 1;
 }
+
+/*
+ * Replication Status Key
+ */
+ message ReplicationStatusKey {
+   enum Status {
+     REPLICATION_COMPLETION = 0;
```

Comment:
Not sure I understand why this is an enum. Probably having as a key the Cluster UUID would make sense? Because at the active, one active can have multiple standby's, so how do we differentiate? And for the standby its ok to have its own cluster id as key. It would look something like this:
 
**Key** <uuid>                              |                      **Value**
Cluster_Standby_Paris_0001        |            ReplicationStatusVal
Cluster_London_002                     |            ReplicationStatusVal